app/src/test/java/com/github/se/cyrcle/model/map/mapViewModelTest.kt

```diff
@@ -0,0 +1,74 @@
+package com.github.se.cyrcle.model.map
+
+import android.annotation.SuppressLint
+import com.github.se.cyrcle.model.map.MapViewModel.LocationPickerState
+import com.github.se.cyrcle.model.parking.Location
+import com.mapbox.geojson.Point
+import com.mapbox.maps.CameraState
+import com.mapbox.maps.EdgeInsets
+import com.mapbox.maps.ScreenCoordinate
+import org.junit.Test
+
+class MapViewModelTest {
+
+  @SuppressLint("StateFlowValueCalledInComposition")
+  @Test
+  fun mapViewModelTestCameraState() {
+    val mapViewModel = MapViewModel()
```

Comment:
you can instantiate a single viewModel for the whole class rather than one per function